_data/cfpb-brand-colors.csv

```diff
@@ -1,59 +1,59 @@
-pacific,pacific,Pacific,#0072ce,"0, 114, 206","90, 48, 0, 0",,true
-pacific,pacific-80,Pacific 80,#4497dc,"68, 151, 220","68, 30, 0, 0",,
-pacific,pacific-60,Pacific 60,#7eb7e8,"126, 183, 232","46, 16, 0, 0",,
-pacific,pacific-40,Pacific 40,#afd2f2,"175, 210, 242","29, 8, 0, 0",,
-pacific,pacific-20,Pacific 20,#d6e8fa,"214, 232, 250","16, 3, 0, 0",,
-navy,dark-navy,Dark Navy,#002d72,"0, 45, 114","100, 80, 6, 32",,
-navy,navy,Navy,#254b87,"37, 75, 135","82, 58, 0, 31",,true
-navy,navy-80,Navy 80,#5674a3,"186, 116, 163","60, 35, 0, 26",,
-navy,navy-60,Navy 60,#889cc0,"136, 156, 192","36, 21, 0, 17",,
-navy,navy-40,Navy 40,#b3c0d9,"179, 192, 217","23, 13, 0, 7",,
-navy,navy-20,Navy 20,#d3daeb,"211, 218, 235","13, 8, 0, 4",,
-purple,dark-purple,Dark Purple,#a01b68,"160, 27, 104","40, 100, 30, 10",,
-purple,purple,Purple,#b4267a,"180, 38, 122","27, 100, 10, 0",,true
-purple,purple-80,Purple 80,#c55998,"197, 89, 152","21, 79, 9, 0",,
-purple,purple-60,Purple 60,#d486b2,"212, 134, 178","14, 58, 8, 0",,
-purple,purple-40,Purple 40,#e3b2cc,"227, 178, 204","9, 36, 6, 0",,
-purple,purple-20,Purple 20,#f0d8e2,"240, 216, 226","4, 15, 2, 0",,
-red,dark-red,Dark Red,#b63014,"182, 48, 20","0, 90, 100, 20",,
-red,red,Red,#d14124,"209, 65, 36","0, 85, 100, 4",,true
-red,red-80,Red 80,#dd735d,"221, 115, 93","0, 75, 75, 0",,
-red,red-60,Red 60,#e79e8e,"231, 158, 142","0, 51, 46, 0",,
-red,red-40,Red 40,#f0c3b8,"240, 195, 184","0, 28, 25, 0",,
-red,red-20,Red 20,#f7e0d9,"247, 224, 217","0, 13, 12, 0",,
-gold,dark-gold,Dark Gold,#dc731c,"220, 115, 28","10, 60, 100, 0",,
-gold,gold,Gold,#ff9e1b,"255, 158, 27","0, 45, 94, 0",,true
-gold,gold-80,Gold 80,#ffb858,"255, 184, 88","0, 34, 76, 0",,
-gold,gold-60,Gold 60,#ffce8d,"255, 206, 141","0, 21, 49, 0",,
-gold,gold-40,Gold 40,#ffe1b9,"255, 225, 185","0, 13, 31, 0",,
-gold,gold-20,Gold 20,#fff0dd,"255, 240, 221","0, 6, 15, 0",,
-neutral,dark-neutral,Dark Neutral,#745745,"116, 87, 69","50, 60, 57, 28",,
-neutral,neutral,Neutral,#8a6c57,"138, 108, 87","50, 60, 57, 7",,true
-neutral,neutral-80,Neutral 80,#a18573,"161, 133, 115","42, 50, 47, 0",,
-neutral,neutral-60,Neutral 60,#baa496,"186, 164, 150","28, 34, 32, 0",,
-neutral,neutral-40,Neutral 40,#d3c5bc,"211, 197, 188","19, 22, 21, 0",,
-neutral,neutral-20,Neutral 20,#e7ddd7,"231, 221, 215","9, 11, 10, 0",,
-gray,black,Black,#101820,"16, 24, 32","0, 0, 0, 100",true,
-gray,dark-gray,Dark Gray,#43484e,"67, 71, 78","0, 0, 0, 87",,
-gray,gray,Gray,#5a5d61,"90, 93, 97","0, 0, 0, 77",,true
-gray,gray-80,Gray 80,#75787b,"117, 120, 123","0, 0, 0, 65",,
-gray,gray-60,Gray 60,#919395,"145, 147, 149","0, 0, 0, 50",,
-gray,gray-40,Gray 40,#b4b5b6,"180, 181, 182","0, 0, 0, 35",,
-gray,gray-20,Gray 20,#d2d3d5,"210, 211, 213","0, 0, 0, 20",,
-gray,gray-10,Gray 10,#e7e8e9,"231, 231, 233","0, 0, 0, 10",,
-gray,gray-5,Gray 5,#f7f8f9,"247, 248, 249","0, 0, 0, 5",,
-gray,white,White,#ffffff,"255, 255, 255","0,  0,  0,  0",,
+family,shortname,name,hex,RGB,PMS,CMYK,primary,secondary,,

```

Comment:
@nataliafitzgerald seems like you need to remove that column in Excel + resave it.